account_lock_date_update/tests/test_account_lock_date_update.py

```diff
@@ -43,5 +43,5 @@ def test_02_update_with_access(self):
             'groups_id': [(4, self.adviser_group.id)],
         })
         wizard.sudo(self.demo_user.id).execute()
-        self.assertEqual(self.company.period_lock_date, '2000-02-01')
-        self.assertEqual(self.company.fiscalyear_lock_date, '2000-01-01')
+        self.assertEqual(str(self.company.period_lock_date), '2000-02-01')
```

Comment:
Better to use `fields.Date.to_string` for avoiding incorrect tests on some systems that don't convert the same format through `str`.

Reply:
Fixed